### ISSUE TITLE
Map not showing on Android with 5.2.0 (RN 0.42.3) - only white

### ISSUE THREAD

**Issue by GeospatialMax** [nitaliano/react-native-mapbox-gl#597]
I have just upgraded to react-native-mapbox-gl 5.2.0 to overcome issues I had in the previous version (5.1.0) that prevented me building for iOS.
Unfortunately now I get a completely white map, independent of which styleURL I set. The map also doesn't seem to have coordinates set correctly.
Every call I do to functions such as getCenterCoordinateZoomLevel or getBounds returns 'undefined'.

Can anyone shine some light on what might be happening here?

To summarize my setup:
react-native-mapbox-gl: 5.2.0
react-native: 0.42.3

**Comment by danieljvdm** [comment]
I'm also getting an completely white screen (sometimes that map flashes for a second while loading) on iOS.

**Comment by GeospatialMax** [comment]
Just for others who experience this problem:
We had opacity set on the map style in 5.1.0 which worked without a problem. After removing the opacity element from the style, the map showed up without an issue.

**Comment by henrikra** [comment]
Please the issue when it is solved :D
